CHANGES.rst

```diff
@@ -3,9 +3,116 @@
 0.14 (unreleased)
```

Comment:
Put the date in this section title, in the same format as previously.

Reply:
Done